jwst/pipeline/calwebb_coron3.py

```diff
@@ -76,44 +74,37 @@ def process(self, input):
             psf_models.append(psf_input)
             psf_input.close()
 
-        # Call the stack_refs step to stack all the PSF images into
-        # a single CubeModel
+        # Stack all the PSF images into a single CubeModel
         psf_stack = self.stack_refs(psf_models)
         psf_models.close()
 
         # Save the resulting PSF stack
-        output_file = mk_prodname(self.output_dir, prod['name'], 'psfstack')
-        log.info('Saving psfstack file %s', output_file)
-        psf_stack.save(output_file)
+        psf_stack.meta.filename = prod['name']
```

Comment:
The original code (`output_file = mk_prodname()`) defines an output filename based on `output_dir` to support writing out the product in another directory.  This seems to be missing from the change where meta.filename is simply set to the rootname without support for specifying an output directory.  Does the `.save_model()` method handle this now in a hidden manner or does this need to be more explicitly defined?  This same question applies for all remaining similar changes...

Reply:
Yes, @stscieisenhamer refactor of `stpipe` put all the path control internally into the pipeline infrastructure.